cmd/whoami/cmd.go

```diff
@@ -140,17 +140,17 @@ func run(_ *cobra.Command, _ []string) {
 		account = response.Body()
 	}
 	fmt.Printf(""+
-		"AWS Account ID:           %s\n"+
-		"AWS Default Region:       %s\n"+
-		"AWS ARN:                  %s\n"+
-		"OpenShift API:            %s\n"+
-		"Account ID:               %s\n"+
-		"Account Name:             %s %s\n"+
-		"Account Username:         %s\n"+
-		"Account Email:            %s\n"+
-		"Organization ID:          %s\n"+
-		"Organization Name:        %s\n"+
-		"Organization External ID: %s\n",
+		"AWS Account ID:               %s\n"+
+		"AWS Default Region:           %s\n"+
+		"AWS ARN:                      %s\n"+
+		"OCM API:                      %s\n"+
```

Comment:
Is this what we want? `OCM Cluster API` maybe?

Reply:
Maybe? What would make sense for customers?

/cc @jeremyeder

Reply:
Let's leave as OCM API for now and we can change later if it causes confusion